packages/strapi-design-system/src/Checkbox/Checkbox.tsx

```diff
@@ -18,40 +16,24 @@ const CheckboxLabel = styled(Typography)<Pick<CheckboxProps, 'disabled'>>`
 `;
 
 const CheckboxTick = React.forwardRef<CheckboxElement, BaseCheckboxProps>((props, forwardedRef) => {
-  const { id } = useField();
+  const id = useId();
```

Comment:
is this needed anymore? If the checkbox has a label already, I don't think it then needs and ID 🤔